src/components/NewClusterModal.js

```diff
@@ -194,9 +197,19 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
       options: _.map(({ label, image }) => ({ label, value: image }), leoImages)
     })
 
-    const isCustomImageInvalid = !imageValidationRegexp.test(customEnvImage)
     const isSelectedImageInputted = selectedLeoImage === CUSTOM_MODE || selectedLeoImage === PROJECT_SPECIFIC_MODE
 
+    const machineTypeConstraints = { inclusion: { within: _.map('name', validMachineTypes), message: 'is not supported' } }
```

Comment:
I don't love this error message, since there's nothing that can be changed marked 'master machine type' (for example). Perhaps if it said something about selecting a new one from a dropdown?